rules-emerging-threats/2023/Malware/GuLoader/proc_creation_win_malware_guloader_execution.yml

```diff
@@ -0,0 +1,27 @@
+title: Injected Browser Process Spawning Rundll32
+id: 89e1490f-1a3e-452a-bbb8-b68a5f58072f
+status: experimental
+description: |
+    Detects the execution of installed GuLoader malware on the host. GuLoader is initiating network connections via the rundll32.exe process that is spawned via a browser parent(injected) process.
+references:
+    - Internal Research
```

Comment:
I put `Internal Research`. But in case you have a reference, that would be better.